src/plugins/discover/public/application/main/discover_main_route.tsx

```diff
@@ -111,7 +104,6 @@ export function DiscoverMainRoute(props: Props) {
 
         const ipList = ip.list;
         const dataViewData = resolveDataView(ip, nextSavedSearch.searchSource, toastNotifications);
```

Comment:
`refreshFields` has been added to the data view loading functions - previously the data view would be loaded, triggering the first field list load, and then it would be loaded again to make sure it was fresh.